randovania/gui/tracker_window.py

```diff
@@ -324,6 +328,13 @@ def _undo_last_action(self):
         self.actions_list.takeItem(len(self._actions))
         self._refresh_for_new_action()
 
+    def _add_manual_action(self):
```

Comment:
if `get_node` becomes `async`, this would also need to be async. You can then use `@asyncSlot()` to solve things